src/core/server/services/users/delete.ts

```diff
@@ -101,12 +109,99 @@ async function deleteUserActionCounts(
   });
 }
 
+async function moderateComments(
+  mongo: Db,
+  redis: AugmentedRedis,
+  tenantID: string,
+  filter: any,
```

Comment:
This can be typed! Add:

```ts
filter: FilterQuery<Comment>,
```

Where `FilterQuery` is from `coral-server/models/helpers`, and `Comment` is the comment model.